### PR TITLE
[CWS] handle kworker loading module

### PR DESCRIPTION
**Pull request by safchain**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by spikat — APPROVED** [review]

File: docs/cloud-workload-security/agent_expressions.md [hunk 1]
```diff
@@ -218,6 +218,7 @@ The *file.rights* attribute can now be used in addition to *file.mode*. *file.mo
 | `process.ancestors.fsuser` | string | FileSystem-user of the process |  |
 | `process.ancestors.gid` | int | GID of the process |  |
 | `process.ancestors.group` | string | Group of the process |  |
+| `process.ancestors.is_kworker` | bool |  |  |
```
Comment (buraizu) [inline docs/cloud-workload-security/agent_expressions.md:221]: Could you confirm if it's expected for these new event types to lack descriptions? All other event types in the table include them.

File: docs/cloud-workload-security/secl.json [hunk 2]
```diff
@@ -313,6 +313,12 @@
           "definition": "Group of the process",
           "constants": ""
         },
+        {
+          "name": "process.ancestors.is_kworker",
+          "type": "bool",
+          "definition": "",
```
Comment (buraizu) [inline docs/cloud-workload-security/secl.json:319]: As above, could you confirm if it's expected for `definition` to be an empty string here, as well as the other new additions to this file?

**Review by buraizu — COMMENTED** [review]
Thanks, left a couple of questions, as it looks like there may be missing values for descriptions and definitions

File: docs/cloud-workload-security/secl.json (hunk 2 above)
Comment (safchain) [reply docs/cloud-workload-security/secl.json:319]: No it wasn't expected, thanks for pointing this out

Comment (safchain) [reply docs/cloud-workload-security/secl.json]: I just updated the PR

**Review by buraizu — APPROVED** [review]

**Pull request merged**